.gitlab-ci.yml

```diff
@@ -820,6 +820,12 @@ library:ci-vst:
   - build:edge+flambda
   - library:ci-flocq
 
+library:ci-deriving:
+  extends: .ci-template-flambda
+  stage: stage-3
+  needs:
+  - library:ci-mathcomp
```

Comment:
```suggestion
  - build:edge+flambda
  - library:ci-mathcomp
```

Reply:
Oh yes, the reason is that as of today gitlab's `needs:` don't pull transitive deps.